src/HtmlGenerator/Smart.Design.Templates/CustomTagHelpersCollection.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Smart.Design.Razor.TagHelpers.Constants;
+
+namespace Smart.Design.RazorView
+{
+    /// <summary>
+    /// </summary>
+    public class CustomTagHelperCollections
+    {
+        public IEnumerable<string> Names => GetNamesFromTypeByReflection(typeof(TagNames))
+            .Select(v => v.GetRawConstantValue() as string);
+
+        private IEnumerable<FieldInfo> GetNamesFromTypeByReflection(Type type)
```

Comment:
No need to mention that it is implemented with reflection in the function name.
```suggestion
        private IEnumerable<FieldInfo> GetNamesFromType(Type type)
```